src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+ const cleanedUpString = sourceString
+    .split('\n')
+    .join('')
+    .trim()
+    .split(';')
+    .map(el => el.split(':'));
+
+  const result = {};
+
+  cleanedUpString.forEach(element => {
+    if (element.length > 1) {
+      const [cssProperty, cssValue] = element;
+
+      const objectKey = cssProperty.trim();
+      const objectValue = cssValue.trim();
+
+      result[objectKey] = objectValue;
+    }
```

Comment:
[CODE PRACTICE] - Instead of using a loop with `forEach`, consider using `reduce` to build the resulting object. This aligns with best practices of using iteration methods and avoids the need for an external `result` object.